pkg/okctlapplication/utils.go

```diff
@@ -0,0 +1,56 @@
+package okctlapplication
+
+import (
+	"os"
+
+	"github.com/oslokommune/okctl/pkg/config"
+	"github.com/oslokommune/okctl/pkg/config/load"
+	"github.com/oslokommune/okctl/pkg/config/state"
+	"github.com/oslokommune/okctl/pkg/okctl"
+	"github.com/spf13/cobra"
+)
+
+func loadRepoData(o *okctl.Okctl, cmd *cobra.Command) error {
```

Comment:
What problem made you replicate this function? It is not entirely unlikely that we are loading user and repository data in a sub-optimal way. If that is the case, we should probably do something about it.

Reply:
I prefer to not fail unless it is impossible not to. I replicated the function so that in the scenario where there is no .okctl available, I'll just populate with dummy data. The alternative (as far as I know) will start the wizard, which will abort the command if a user chooses to cancel the wizard

Reply:
Is this because asking for user input messes up the piping of data?

Reply:
Mostly because I by default don't want to force the user into some kind of pattern. Might be an erronous assumption in this case considering our opinionated approach, I just find it a neat rule to live by.